schema/string.go

```diff
@@ -17,7 +17,19 @@ const (
 	stringUUIDVersion = 4
 )
 
-func decodeString(format, value string) (string, error) {
+func checkStringConstraints(v string, minLength int, t string) error {
```

Comment:
This is a more personal comment (I won't block the PR due to this), but I believe it would be clearer if you inline this method within decodeString. An example of this kind of change would be: https://github.com/frictionlessdata/tableschema-go/blob/master/schema/field.go#L120

Reply:
I believe I made a mistake when marking the review and this comment became outdated. Just make sure that stays under your radar. Thanks!